src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -152,5 +159,20 @@ public AddressBook load() throws StorageOperationException {
     public String getPath() {
         return path.toString();
     }
-
+    
+    public void createAddressBookFile(AddressBook addressBook) throws StorageOperationException {
```

Comment:
this will cater to only one case of the problem, you need to make it generic (for all command calls)